### ISSUE TITLE
Not compatible with skia > 1.0

### ISSUE THREAD

**Issue by RRaideRR** [mateoguzmana/react-native-fiesta#38]
**Describe the bug**
The library is not compatible with @shopify/react-native-skia > 1.0

**To Reproduce**
Install the latest version of @shopify/react-native-skia - e.g. 1.2.3

Play a basic animation - e.g.

```
<Fireworks
      numberOfFireworks={5}
      fireworkRadius={200}
      autoHide={true}
      particleRadius={8}
      theme={['red', 'green', 'yellow']}
    />
```

This will throw an error: ` ERROR  TypeError: Cannot read property 'linear' of undefined`

**Expected behavior**
The fireworks animation is played.

**Additional context**
I think the library would need a rewrite to work with skia > 1.0. [As pointed out in the skia repo](https://github.com/Shopify/react-native-skia/issues/2115#issuecomment-1879983880), fiesta would need to migrate to reanimated3. 

I see that this probably requires a lot of work. As an alternative: is it possible to set skia as an explicit dependency for fiesta with a version < 1.0? But this would probably bloat up the size of fiesta - right?

**Comment by mateoguzmana** [comment]
Hey there @RRaideRR, thanks for reporting the issue.

It's been a while since the library deps haven't been updated but I am planning to do it soon in order to support the latest version of Skia. Will leave this issue open to keep track of it.

**Comment by mateoguzmana** [comment]
Support for `@shopify/react-native-skia` 1.x.x and reanimated 3 is now available in the package version 0.6.0
